### ISSUE TITLE
NullPointerException: crash opening a .zip file after a download

### ISSUE THREAD

**Issue by Massedil** [zhanghai/MaterialFiles#1242]
```
FATAL EXCEPTION: main
Process: me.zhanghai.android.files, PID: 4365
java.lang.NullPointerException
	at Z4.g.b(SourceFile:19)
	at Z4.g.getParent(SourceFile:1)
	at L4.k.a(SourceFile:32)
	at L4.K0.u(SourceFile:6)
	at me.zhanghai.android.files.filelist.FileListFragment.z(SourceFile:920)
	at androidx.fragment.app.a.a(SourceFile:52)
	at androidx.fragment.app.a.k(SourceFile:139)
	at g0.O.z(SourceFile:1063)
	at g0.O.Q(SourceFile:82)
	at g0.O.x(SourceFile:78)
	at g0.O.t(SourceFile:67)
	at g0.B.onStart(SourceFile:34)
	at h.p.onStart(SourceFile:1)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1645)
	at android.app.Activity.performStart(Activity.java:8800)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3971)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:299)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:279)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleItem(TransactionExecutor.java:251)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:228)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:91)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2544)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8501)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
```

**Comment by Massedil** [comment]
In Fennec F-Droid go to the download menu and click on a just downloaded .zip file. MaterialFiles crashes. If you open the same .zip file from MaterialFiles, it works fine.

**Comment by FooIbar** [comment]
Fixed but not released yet, see #1225
